pkg/object/meshcontroller/worker/egress.go

```diff
@@ -226,6 +234,13 @@ func (egs *EgressServer) reloadByTrafficTargets(value map[string]*spec.TrafficTa
 	return true
 }
 
+func (egs *EgressServer) reloadByServiceCanaries(value map[string]*spec.ServiceCanary) bool {
+	select {
+	case egs.chReloadEvent <- struct{}{}:
+	}
```

Comment:
```suggestion
	egs.chReloadEvent <- struct{}{}
```

Reply:
Got it. My static checker just report this. FYI, https://staticcheck.io/docs/checks#S1000 "should use a simple channel send/receive instead of select with a single case (S1000) go-staticcheck".

Reply:
Updated with correct select statement